### PR TITLE
Forget password form

### PR DESCRIPTION
**Pull request by bitspook**

### CODE REVIEW

File: ui/src/components/ForgetPassword/style.module.scss [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+  font-weight: 500;
+  font-size: 14px;
+  margin-bottom: 32px;
+  border-radius: 4px;
+  border: none;
+}
+
+.title-container{
+  /* padding: 46px; */
+}
```
Comment (bitspook) [inline ui/src/components/ForgetPassword/style.module.scss]: Looks like we can remove this.

File: ui/src/components/LoginForm/index.tsx [hunk 2]
```diff
@@ -38,29 +42,43 @@ const Login: React.FC<FormikProps<LoginFormValues>> = (props) => {
...
+                className={cx('email')}
+                placeholder="> enter your work email"
+                type="email"
+                name="email"
+              />
 
-          <div className={cx('password')}>
-            <span className={cx('back-icon')} onClick={gotoStep1} />
+              <div className={cx('password')}>
+                <span className={cx('back-icon')} onClick={gotoStep1} />
```
Comment (bitspook) [inline ui/src/components/LoginForm/index.tsx:64]: Looks like this isn't formatted properly.

File: ui/src/components/LoginForm/style.module.scss [hunk 3]
```diff
@@ -8,6 +8,16 @@
...
 }
 
+.login-form-container {
+  @extend %animate;
+
+  animation-name: undim-sliding-down;
+  animation-delay: 0.6s;
+  margin-top: 3em;
+
+  // margin-bottom: 8em;
```
Comment (bitspook) [inline ui/src/components/LoginForm/style.module.scss]: Please don't leave commented out code behind.

File: ui/src/components/LoginForm/index.tsx (hunk 2 above)
Comment (jkhabra) [reply ui/src/components/LoginForm/index.tsx:64]: ok

**Pull request merged**